docs/quickstart/quickstart_chains/arbitrum.md

```diff
@@ -0,0 +1,305 @@
+              # address: "0x60781C2586D68229fde47564546784ab3fACA982"
+```
+
+
+The above code indicates that you will be running a `handleDividendBatch` mapping function whenever there is a `ClaimDividendBatch` log on any transaction from the [WINR contract](https://arbiscan.io/address/0xddaecf4b02a3e45b96fc2d7339c997e072b0d034#code).
+
+Check out our [Manifest File](../../build/manifest/ethereum.md) documentation to get more information about the Project Manifest (`project.yaml`) file.
+
+## 2. Update Your GraphQL Schema File
+
+The `schema.graphql` file determines the shape of your data from SubQuery due to the mechanism of the GraphQL query language. Hence, updating the GraphQL Schema file is the perfect place to start. It allows you to define your end goal right at the start.
+
+Remove all existing entities and update the `schema.graphql` file as follows. Here you can see we are indexing block information such as the id, blockHeight and timestamp along with the user, the total rewards and the dividends.
+
+
+```graphql
+type Dividend @entity {
+  id: ID! # Transaction hash
+  blockHeight: BigInt!
+  timestamp: BigInt!
+  user: User!
+  reward: BigInt!
+}
+
+type User @entity {
+  id: ID! # Transaction hash
+  totalRewards: BigInt!
+  dividends: [Dividend]! @derivedFrom(field: "user")
+}
+}
+```
+
+::: warning Important
+When you make any changes to the schema file, please ensure that you regenerate your types directory.
+:::
+
+SubQuery makes it easy and type-safe to work with your GraphQL entities, as well as smart contracts, events, transactions, and logs. SubQuery CLI will generate types from your project's GraphQL schema and any contract ABIs included in the data sources.
+
+::: code-tabs
+@tab:active yarn
+
+```shell
+yarn codegen
+```
+
+@tab npm
+
+```shell
+npm run-script codegen
+```
+
+:::
+
+This will create a new directory (or update the existing one) `src/types` which contains generated entity classes for each type you have defined previously in `schema.graphql`. These classes provide type-safe entity loading, and read and write access to entity fields - see more about this process in [the GraphQL Schema](../../build/graphql.md). All entities can be imported from the following directory:
+
+```ts
+import { Dividend, User } from "../types";
+```
+
+If you're creating a new Etheruem based project, this command will also generate ABI types and save them into `src/types` using the `npx typechain --target=ethers-v5` command, allowing you to bind these contracts to specific addresses in the mappings and call read-only contract methods against the block being processed. It will also generate a class for every contract event to provide easy access to event parameters, as well as the block and transaction the event originated from. All of these types are written to `src/typs/abi-interfaces` and `src/typs/contracts` directories. In the example Gravatar SubQuery project, you would import these types like so.
```

Comment:
typo in `src/typs/contracts`

Reply:
best search to see if this typo is elsewhere

Reply:
"In the example Gravatar SubQuery project, you would import these types like so." - needs updating

Reply:
Fixed.